runtime/src/launch/slurm-gasnetrun_common/slurm-gasnetrun_common.h

```diff
@@ -173,17 +173,13 @@ static void genNumLocalesOptions(FILE* slurmFile, sbatchVersion sbatch,
   }
 }
 
-static int propagate_environment(char* buf, size_t size)
+static void propagate_environment(char** buf)
 {
-  int len = 0;
-
   // Indiscriminately propagate all environment variables.
   // We could do this more selectively, but we would be likely
   // to leave out something important.
   char *enviro_keys = chpl_get_enviro_keys(',');
-  if (enviro_keys)
-    len += snprintf(buf, size, " -E '%s'", enviro_keys);
-  return len;
+  if (enviro_keys) chpl_append_to_cmd(buf, " -E '%s'", enviro_keys);
```

Comment:
This missing the `numWritten` argument to `chpl_append_to_cmd`

Reply:
Oof, and also `buf` is a single pointer rather than double as required. I guess the gasnet run I did does not exercise this path.